.travis.yml

```diff
@@ -4,7 +4,7 @@ sudo: false
 cache: bundler
 
 before_install:
-  - gem update --system
+  - rvm rubygems latest
```

Comment:
How does this fixes Travis? o_O Not saying it's not, just trying to understand why.

Reply:
The issue here can be seen : https://travis-ci.org/httprb/http/jobs/625515097

```
$ gem update --system
Updating rubygems-update
Fetching rubygems-update-3.1.0.gem
Successfully installed rubygems-update-3.1.0
Installing RubyGems 3.1.0
  Successfully built RubyGem
  Name: bundler
  Version: 2.1.0.pre.3
  File: bundler-2.1.0.pre.3.gem
bundler's executable "bundle" conflicts with /home/travis/.rvm/rubies/ruby-2.4.6/bin/bundle
Overwrite the executable? [yN]  
No output has been received i
```

which is related to : https://github.com/rubygems/rubygems/issues/3030

Now looking more carefully, it seems that the change didn't had the effect I wanted, but worked because it basically did a no-op here.

Reply:
There seems to be a better workaround here: https://github.com/rubygems/rubygems/issues/3036 (I will amend the commit)